one-pager-maker/src/App.tsx

```diff
@@ -36,18 +36,29 @@ function App() {
             <BrowserRouter>
                 {user ? (
                     <>
-                        <nav>
-                            <ul id="nav">
-                                <li>
-                                    <Link to="/">一覧</Link>
-                                </li>
-                                <li>
-                                    <Link to="/edit">編集</Link>
-                                </li>
-                                <li>
-                                    <Button onClick={() => { signOut(auth) }}>ログアウト</Button>
+                        <nav className="relative px-4 py-4 flex justify-between items-center bg-white">
+                            <Link className="text-3xl font-bold leading-none" to="/">
+                                <FiFileText className='inline' />
+                                <a className="p-1">One Pager Maker</a>
+                            </Link>
+                            <div className="lg:hidden">
+                                <button className="navbar-burger flex items-center text-blue-600 p-3">
+                                    <svg className="block h-4 w-4 fill-current" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+                                        <title>Mobile menu</title>
+                                        <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"></path>
+                                    </svg>
+                                </button>
+                            </div>
+                            <ul className="hidden absolute top-1/2 left-1/2 transform -translate-y-1/2 -translate-x-1/2 lg:flex lg:mx-auto lg:flex lg:items-center lg:w-auto lg:space-x-6">
+                                <li><Link className="text-sm text-gray-400 hover:text-gray-500" to="/">Home</Link></li>
```

Comment:
The same Link component appears below as "About" link, so you can make it independent component.

Reply:
I don't quite understand this point, so please explain a little more.

Reply:
Do you mean make Link Button a Component?

Reply:
yes, I felt it was redundant to declare nearly same <Link/> component

Reply:
ah. It's the same except for the title. understood. I made it a component.